docs/guides/runtime-options-overview.mdx

```diff
@@ -135,11 +137,16 @@ Scroll to see the full table.
 |                      | num_randomizations       |                         |  `auto`/ integer >= 1                                                                        | `'auto'`                    |                    |
 |                      | shots_per_randomization  |                         |   `auto`/ Integer no larger than `backend.max_shots`.                                        | `'auto'`                    |                    |
 |                      | strategy                 |                         | `'active'`<br />`'active-circuit'`<br />`'active-accum'`<br />`'all'`                        | `'active-accum'`            |                    |
+| experimental         |                          |                         |                           |                        |                    |
+|                      | execution_path           |                         | `gen3-turbo` *([see note](#experimental-option-note))*               | None                   |                    |
 
   </TabItem>
 </Tabs>
 
-
+<span id="experimental-option-note"></span>
+<Admonition type="note">
+Setting `gen3-turbo` enables the next-generation execution path. This is experimental and is only available on `ibm_torino`.
```

Comment:
```suggestion
Setting `gen3-turbo` enables the next-generation execution path, which provides a great improvement in performance. This is experimental and is only available on `ibm_torino`.
```

Let's give them a little teaser on what's coming 😄. 
In the API reference, we point out experimental options are "subject to change without notification, and stability is not guaranteed". Do you think we need to mention it again here? Or would that make the sentence too long?